.github/workflows/release-packages.yaml

```diff
@@ -217,11 +217,10 @@ jobs:
           submodules: recursive
       - name: Set Image Tag
         run: |
-          echo ${{ github.ref }}
-          echo $GITHUB_REF
-          echo $GITHUB_REF | cut -d "/" -f 3 >> $VERSION
-          arrVERSION=(${VERSION//-/ })
-          echo "IMAGE_TAG=diffblue/cbmc:${arrVERSION[1]}" >> $GITHUB_ENV
+          # Isolate the version number from a reference to a tag, for example,
+          # '5.20.3' from a string like 'refs/tags/cbmc-5.20.3-exp'
+          VERSION=$(echo ${{ github.ref }} | cut -d "/" -f 3 | cut -d "-" -f 2)
```

Comment:
is the tag reference always in that form? I wonder if that can go wrong in the future if tag is placed under different path

Reply:
Maybe use `basename`? 
something like:
`VERSION=$(basename ${{ github.ref }}  | cut -d "-" -f 2)`

Reply:
This is related to git internals, so I doubt it's going to change anytime soon https://git-scm.com/book/en/v2/Git-Internals-Git-References

Reply:
@piotr-grabalski As to the tagname starting with `refs/tags/`, yes, it'll be in that form. This is how it's handled in git and unlikely to change, certainly not without a major version change in git itself. As to the tagname itself, this should always be `cbmc-$VERSION`, if we're changing that of course we need to change our scripts but I'm not sure why we would change it.